test/src/blockMonitor/blockchainMachine.test.ts

```diff
@@ -143,7 +143,7 @@ describe("BlockchainMachine", () => {
         bm.addComponent(new ExampleComponent(reducer));
         await bm.start();
 
-        await blockCache.addBlock(blocks[0]);
+        await blockStore.withBatch(async () => await blockCache.addBlock(blocks[0]));
```

Comment:
Again, i think these tests should be using a mock blockcache.

Reply:
As discussed, opened issue #338 for this.